src/fides/api/cryptography/cryptographic_util.py

```diff
@@ -13,7 +13,7 @@ def decode_password(password: str) -> str:
     """
     try:
         return b64_str_to_str(password)
-    except Error:
+    except (Error, UnicodeDecodeError):
```

Comment:
decided to keep this minimal in scope - only catching the error i know can be thrown in the error case. handling more generic errors/exceptions would be more likely to lead to unintended side effects, IMO.

also didn't add any sort of logging here as it could get very noisy, but i'd be open to an argument for that